distribution/changelog.txt

```diff
@@ -24,6 +24,7 @@
 - Fix: [#21198] [Plugin] Setting brake or booster speeds on a tile element doesn’t work.
 - Fix: [#21290] Sound keeps playing when paused from fast-forward mode.
 - Fix: [#21291] Hungry guests heading to any flat ride do not count for warning threshold (original bug).
+- Fix: [#21309] Africa - Oasis & Blackpool Pleasure Beach's monorails travel out of bounds.
```

Comment:
Use curly quotes, rather than typewriter quotes.

```suggestion
- Fix: [#21309] Africa - Oasis & Blackpool Pleasure Beach’s monorails travel out of bounds.
```